linera-views/tests/random_container_tests.rs

```diff
@@ -370,10 +370,14 @@ async fn run_map_view_mutability<R: RngCore + Clone>(rng: &mut R) -> Result<()>
                 let part_key_values = view.map.key_values_by_prefix(vec![u]).await?;
                 assert_eq!(part_state_vec, part_key_values);
             }
-            for key in &all_keys {
+            let keys_vec = all_keys.clone().into_iter().collect::<Vec<_>>();
```

Comment:
This should avoid allocating and then consuming another `BTreeSet`:

```suggestion
            let keys_vec = all_keys.iter().cloned().collect::<Vec<_>>();
```

Reply:
Ok.